plugins/inputs/kernel/sample.conf

```diff
@@ -1,4 +1,12 @@
-# Get kernel statistics from /proc/stat
+# Get various kernel statistics.
+# Currently collects:
+#   /proc/stat
+#   /proc/entropy
+#   /sys/kernel/mm/ksm (optional, if enabled)
+#
```

Comment:
Can you please only add a short description here and add the details to the general README? The reason is that this will also end-up in the example configuration file which is quite long already. How about
```suggestion
# Plugin to collect various Linux kernel statistics.
```